provisionersdk/proto/provisioner.proto

```diff
@@ -106,8 +106,9 @@ message App {
     string url = 4;
     string icon = 5;
     bool subdomain = 6;
-    Healthcheck healthcheck = 7;
-    AppSharingLevel sharing_level = 8;
+    bool external = 7;
+    Healthcheck healthcheck = 8;
+    AppSharingLevel sharing_level = 9;
```

Comment:
I don't think you're supposed to reorder proto fields, since it won't be backwards compat. I'd just add `external` as 9.

Reply:
Fixed!